toolchain/check/import_cpp.cpp

```diff
@@ -2018,11 +2018,113 @@ auto ImportNameFromCpp(Context& context, SemIR::LocId loc_id,
 }
 
 static auto GetOperatorKind(Context& context, SemIR::LocId loc_id,
```

Comment:
nit: maybe GetClangOperatorKind?

I guess this function may be moving into a subdir soon though, so maybe into a namespace? 

But something to delineate it as c/c++ stuff would be nice

Reply:
Done.
Note that this is a static function so its already internal to the `cpp` named file so a namespace would not add much for this specific function.